mgmt/rest/client/client_func_test.go

```diff
@@ -495,4 +508,10 @@ func TestSnapClient(t *testing.T) {
 		So(p2.Err, ShouldNotBeNil)
 		So(p3.Err, ShouldNotBeNil)
 	})
+
+	c, err = New("test", "", true)
```

Comment:
Tests should be expanded to more than just testing passing a string. Will the client error if someone passes in `htp://` or `ftp://`, etc?